datagovharvester/extract/dcatus.py

```diff
@@ -0,0 +1,57 @@
+from .utils import download_catalog
+import json
+from datagovharvester.utils.s3_utilities import (
+    upload_to_S3,
+    create_s3_payload,
+)
+
+def store_record_in_s3( record, source_id, job_id, record_idx, S3_client, 
+    key_name, bucket_name ):
+    """ store the input record in a s3 bucket
+    record (dict)               :   dcatus record
+    source_id (str)             :   uuid 
+    job_id (str)                :   uuid
+    record_idx (int)            :   index position of record in records list
+    S3_client (boto3.client)    :   boto3 s3 client. 
```

Comment:
Looks like you intended to include `key_name` here? It's probably worth adding since you return it, also

Reply:
I ended up nixing this function but good find! I'm gonna double check my docstrings to make sure i'm including everything.